cloudbuild.yaml

```diff
@@ -0,0 +1,31 @@
+steps:
```

Comment:
Not sure of the best way to organize these config files. Would it make sense to create a `config` subdir?

Reply:
For small projects/repositories like this, they are typically left in the project root to simplify automation. We might be able to de-clutter 3-5 files into a separate directory, but we'd need to add more build scripts.